13.0 database block (#238)
itpp-labs/access-addons: database_block/README.rst

.. image:: https://img.shields.io/badge/license-MIT-blue.svg
   :target: https://opensource.org/licenses/MIT
   :alt: License: MIT

================

>  Database Block

Reviewer: Block backend UI

?

================

> This module allows to block user to backend with specified reason

Reviewer: ```suggestion
This module allows blocking backend access and display the message
```
